src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj

```diff
@@ -24,6 +31,8 @@
 		<PackageReference Include="Uno.Core" Version="4.0.1" />
 		<PackageReference Include="Uno.UI" Version="4.0.13" />
 		<PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0" />
+		<!--Microsoft.SourceLink.GitHub is needed for Source Link support -->
```

Comment:
I think you need more than that to configure Source Link.
Here's a full example: https://github.com/nventive/Chinook.DataLoader/pull/36/files
Adding Source Link also seems out of scope.

Reply:
I'm not sure we need the workaround anymore. Can you try pack locally @carlh98 and check that the nupkg is fine in your nuget explorer ?

Reply:
I've verified the packages locally in the nuget explorer